codegen/generator/nodejs/templates/functions.go

```diff
@@ -41,9 +41,24 @@ var (
 		"ToLowerCase":         commonFunc.ToLowerCase,
 		"ToUpperCase":         commonFunc.ToUpperCase,
 		"ToSingleType":        toSingleType,
+		"ListOfEnum":          listOfEnum,
 	}
 )
 
+func listOfEnum() string {
+	schema := generator.GetSchema()
+
+	var result []string
+
+	for _, t := range schema.Types {
+		if t.Kind == introspection.TypeKindEnum && !strings.HasPrefix(t.Name, "__") {
```

Comment:
Curious: what is the `__` prefix handling for? Might benefit from being split out into a function whose name answers that question.

Reply:
There are few internals enums with `__` prefix that we should ignore